com.microsoft.mrtk.accessibility/Utilities/DescribableObject.cs

```diff
@@ -12,6 +12,15 @@ namespace Microsoft.MixedReality.Toolkit.Accessibility
     [AddComponentMenu("MRTK/Accessibility/Describable Object")]
     public class DescribableObject : MonoBehaviour
     {
+        [SerializeField]
+        [Tooltip("What is the classification (ex: person, place, ui element, etc.) is this object?")]
+        private ObjectClassification classification = (ObjectClassification)0;
```

Comment:
We should create a direct `None` classification rather than implicitly casting an int.

Reply:
Unity creates the None classification since these are flags.